utility/src/main/java/org/oppia/util/data/HtmlParser.kt

```diff
@@ -0,0 +1,38 @@
+package org.oppia.util.data
+
+import android.content.Context
+import android.text.Html
+import android.text.Spannable
+import android.widget.TextView
+
+private const val CUSTOM_TAG = "oppia-noninteractive-image"
+private const val HTML_TAG = "img"
+private const val CUSTOM_ATTRIBUTE = "filepath-with-value"
+private const val HTML_ATTRIBUTE = "src"
+
+/** Html Parser for android TextView to parse Html tag. */
+class HtmlParser(internal var context: Context) {
```

Comment:
Do we need ``Context``? It appears unused.

Reply:
It is used when UrlImageParser is available.